src/components/Uploader.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import firebase from 'firebase';
+import FileUploader from 'react-firebase-file-uploader';
+
+class Uploader extends Component {
+  state = {
+    mediaPaths: [],
+  };
+
+  handleChangeImage = e => {
+    const { target: { files } } = e;
+    const { getMedia } = this.props;
+    const filesToStore = [];
+
+    for (let key in files) {
+      let file = files[key];
+      filesToStore.push(file);
+    }
+
+    getMedia(filesToStore, this.fileUploader);
+
+  };
+
+  handleUploadError = error => {
+    console.error(error);
+  };
+
+  handleUploadSuccess = async filename => {
+    const { passPaths } = this.props;
+    const { mediaPaths } = this.state;
+    await firebase
```

Comment:
So when we're making GET requests to firebase, should we be doing it like this?

Or is it fine to get the url from process.env and just do a `fetch(...)` call?

Reply:
I think that you'll just want to use `axios.get()` like I did with the form submission. This method chaining is something that's specific for getting the storage URLs.